src/strategies/farm-unilp/index.ts

```diff
@@ -0,0 +1,33 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+import abi from './NoMintRewardPool.json';
+import { Interface } from '@ethersproject/abi';
+import { Contract } from '@ethersproject/contracts';
+import uniLpAbi from './UniLPABI.json';
+
+const UNISWAP_MINT_CONTRACT = '0x514906FC121c7878424a5C928cad1852CC545892';
```

Comment:
Your strategy should be made generic enough so it could be used by others. You shouldn't hardcode an address here you can use the strategy options for that. There is another PR here that overlap with what you are doing https://github.com/bonustrack/snapshot.js/pull/7 Also i think a better approach would be to do it this way: https://github.com/bonustrack/snapshot.js/pull/7#issuecomment-703250682